test/index.spec.js

```diff
@@ -18,9 +18,14 @@ describe('totp generation', () => {
 		expect(totp('JBSWY3DPEHPK3PXP')).toEqual('089029');
 	});
 
-	it('should generate token with date now = 2016', () => {
```

Comment:
Note that I removed the original test here because it's a duplicate of the one on line 11.